scss/_patterns_subnav.scss

```diff
@@ -178,11 +190,12 @@
   // but vf-url-friendly-color doesn't support rgba(),
   // so we are using #999 as fallback for now
   @include vf-subnav-theme(
-    $color-subnav-icon: #999,
     $color-subnav-background: $colors--dark-theme--background-alt,
+    $color-subnav-background-hover: $colors--dark-theme--background-hover,
+    $color-subnav-background-active: $colors--dark-theme--background-active,
     $color-subnav-text: $colors--dark-theme--text-default,
-    $color-subnav-text-hover: $colors--dark-theme--background-default,
-    $color-subnav-background-hover: $colors--dark-theme--text-default,
+    $color-subnav-text-hover: $colors--dark-theme--text-default,
+    $color-subnav-icon: #999,
```

Comment:
Why hardcoding the color without any variable?

Reply:
fixed